### PR TITLE
Update Header and cleanup

### PR DESCRIPTION
**Pull request by eneufeld**
Set correct headers to all files.

### CODE REVIEW

File: backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java [hunk 1]
```diff
@@ -1,3 +1,13 @@
+/*******************************************************************************
```
Comment (lucas-koehler) [inline backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java:1]: There are two copyright headers now :(

File: web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts [hunk 2]
```diff
@@ -77,21 +72,21 @@ export function registerDSL(): void {
...
-        ],
+        // operators: [
+        //     ':', '=', ','
+        // ],
 
         // we include these common regular expressions
-        symbols: /[=><!~?:&|+\-*/^%]+/,
-        escapes: /\\(?:[abfnrtv\\"']|x[0-9A-Fa-f]{1,4}|u[0-9A-Fa-f]{4}|U[0-9A-Fa-f]{8})/,
+        // symbols: /[=><!~?:&|+\-*/^%]+/,
+        // escapes: /\\(?:[abfnrtv\\"']|x[0-9A-Fa-f]{1,4}|u[0-9A-Fa-f]{4}|U[0-9A-Fa-f]{8})/,
```
Comment (lucas-koehler) [inline web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts:89]: Why is everything commented out now?

**Review by lucas-koehler — CHANGES_REQUESTED** [review]
The new copyright header generally looks good to me. However, the files in `backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph` now have two copyright headers. I commented on one of them as an example.

File: backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java (hunk 1 above)
Comment (eneufeld) [reply backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java:1]: I'll check thank you

File: web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts (hunk 2 above)
Comment (eneufeld) [reply web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts:89]: eslint complains that this is not valid

Comment (lucas-koehler) [reply web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts:89]: Okay but wouldn't just removing this basically break the workflow DSL text editor? As I understand it, this defines the DSL and all syntax highlighting, etc will then be gone there. Or am I mistaken?

Comment (eneufeld) [reply web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts:89]: good point, I see what I can do

File: backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java (hunk 1 above)
Comment (lucas-koehler) [reply backend/plugins/com.eclipsesource.workflow.glsp.server/src/main/java-gen/com/eclipsesource/workflow/glsp/server/wfgraph/Icon.java:1]: I did some quick research and found out that fields like `keywords` are not part of the interface but are even used in the official examples. So maybe use `as any` or extend `monaco.languages.IMonarchLanguage`.

File: web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts (hunk 2 above)
Comment (lucas-koehler) [reply web/coffee-workflow-analyzer-editors/src/browser/language-contribution.ts:89]: I did some quick research and found out that fields like keywords are not part of the interface but are even used in the official examples. So maybe use as any or extend monaco.languages.IMonarchLanguage.

**Review by lucas-koehler — APPROVED** [review]

**Pull request merged**